### PR TITLE
[WIP] Add KDE PIM ebuild

### PR DESCRIPTION
**Pull request by a17r**
Monolithic ebuild as a base to split off all the parts.

### CODE REVIEW

File: kde-apps/kdepim/kdepim-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+PATCHES=(
+	"${FILESDIR}/${PN}-revert-qwebengineview.patch"
+	"${FILESDIR}/${PN}-find-gpgme.patch"
+)
+
+src_prepare() {
+	kde5_src_prepare
+
+	cp "${FILESDIR}/FindGpgme.cmake" "${S}"/cmake/modules
```
Comment (johu) [inline kde-apps/kdepim/kdepim-9999.ebuild]: I would add FindGpgme.cmake to the find-gpgme.patch, it is easier to upstream the patch  then.

Comment (a17r) [inline kde-apps/kdepim/kdepim-9999.ebuild]: Maybe it should be added to ECM anyway, I've taken the file from kwalletd.

**Comment by johu** [comment]
@a17r  Is there any progress on upstreaming the patches? To you talked with upstream about the repo split?

**Comment by a17r** [comment]
The information I got was that gpgme should be found through gpgmepp, and upstream would be looking into it, but I still needed the patch a few days ago. Need to get back to upstream, but I should do that from my plasma5 system.

**Comment by johu** [comment]
If we go with this solution we need a make.defaults with USE_EXPAND and the default enabled features.

**Comment by a17r** [comment]
Pushed the latest state. Almost all the apps build independently of each other, except for kontact (which isn't of any use by itself anyway so I just forced kmail in that case) and kalarm which either needs the same treatment or get its kmail dependency made optional.

**Comment by johu** [comment]
Thanks for your work. We will go with the monolithic approach for now, as we don't  need to re-introduce the eclass hacks for splitted version from kde:4 times.

**Pull request merged**